src/renderer.ts

```diff
@@ -119,9 +119,7 @@ export class Renderer {
         //   (the script gets cached by electron the first time it's used so this isn't a particularly high priority)
         const htmlHead = `<script src="https://www.desmos.com/api/v1.6/calculator.js?apiKey=dcb31709b452b1cf9dc26972add0fda6"></script>`;
         const htmlBody = `
-            <div id="calculator-${hash}" style="width: ${graphSettings.width}px; height: ${
-            graphSettings.height
-        }px;"></div>
+            <div id="calculator-${hash}" style="width: 100%; height: ${graphSettings.height.value + graphSettings.height.unit};"></div>
```

Comment:
Forcing "width: 100%" is unnecessary. You seem to be editing the width of the element, then overwriting the change immediately.